README.md

```diff
@@ -20,37 +23,31 @@ including our GPG key, can be found at https://www.influxdata.com/how-to-report-
 
 1. [**Clone this repository**](https://help.github.com/articles/cloning-a-repository/) to your local machine.
 
-2. **Install Hugo**
-
-    The InfluxData documentation uses [Hugo](https://gohugo.io/), a static site generator built in Go.
-    The InfluxData documentation utilizes Hugo's asset pipeline and requires the extended version of Hugo.
-    See the Hugo documentation for information about how to [download and install Hugo](https://gohugo.io/getting-started/installing/).
+2.  **Install NodeJS, Yarn, Hugo, & Asset Pipeline Tools**
 
-    _**Note:** The most recent version of Hugo tested with this documentation is **0.81.0**._
+    The InfluxData documentation uses [Hugo](https://gohugo.io/), a static site generator built in Go.   The site utilizes Hugo's asset pipeline, requiring the extended version of Hugo along with NodeJS tools like PostCSS, to build and process stylesheets and javascript.
 
-3.  **Install NodeJS, Yarn, & Asset Pipeline Tools**
-
-    This project uses tools written in NodeJS to build and process stylesheets and javascript.
-    To successfully build assets:
+    To install all the required dependencies, including hugo-extended, and then build the assets:
 
     1. [Install NodeJS](https://nodejs.org/en/download/)
     2. [Install Yarn](https://classic.yarnpkg.com/en/docs/install/)
-    3. Run the following command to install the necessary tools:
+    3. Install dependencies:
 
-      ```sh
-      sudo yarn global add postcss-cli@8.3.0 autoprefixer@9.8.6
-      ```
+          ```
+          $ cd docs-v2
+          $ yarn install
+          ```
```

Comment:
not sure if it makes sense to tell folks to delete package.json, but I get the following error (which may be trivial?) when testing:
```kseivert@MBP15-KSEIVERT docs-v2 % yarn install
yarn install v1.22.10
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```

Reply:
This is trivial. This just means that you likely used npm to generate assets in your local repo. For fresh clones of the docs repo, this won't matter.